backend/src/routes/api/components/list.ts

```diff
@@ -45,3 +30,36 @@ export const listComponents = async (
 
   return installedComponents;
 };
+
+export const removeComponent = async (
+  fastify: KubeFastifyInstance,
+  request: FastifyRequest,
+): Promise<{ success: boolean; error: string }> => {
+  const query = request.query as { [key: string]: string };
+  const coreV1Api = fastify.kube.coreV1Api;
+  const enabledAppsConfigMapName = 'odh-enabled-applications-config';
```

Comment:
This should probably be different for upstream vs downstream. Either define a constant in `constants.ts` so it is easier to change w/o merge conflicts or (better) in the `.env` file.